pkg/columns/columns.go

```diff
@@ -198,6 +198,15 @@ func (c *Columns[T]) iterateFields(t reflect.Type, sub []int) error {
 			return fmt.Errorf("error parsing tag for %q on field %q: %w", t.Name(), f.Name, err)
 		}
 
+		// fall back to struct field name if column name is empty
+		if column.Name == "" {
+			column.Name = f.Name
+		}
+
+		if column.MinWidth > column.Width {
+			column.Width = column.MinWidth
+		}
```

Comment:
I would prefer this to return an error.

Reply:
This also applies when no width was set (default being used), but I can add an error specifically for the case that both minWidth and width have been set and then minWidth is > width.